### PR TITLE
Fix bug with registration email not sending

### PR DESCRIPTION
**Pull request by Icidis**
User registration email notifications not being sent when Portal registration set as NoRegistration

Fix bug #173

### CODE REVIEW

File: Extensions/Manage/Dnn.PersonaBar.Users/Components/RegisterController.cs
Comment (sleupold) [inline Extensions/Manage/Dnn.PersonaBar.Users/Components/RegisterController.cs:287]: Did you verify, MessageType.UserRegistrationPublic contains the proper message text?

**Review by sleupold — COMMENTED** [review]
I'd prefer this to be moved to the library instead of sending it directly from the UI

File: Extensions/Manage/Dnn.PersonaBar.Users/Components/RegisterController.cs
Comment (Icidis) [reply Extensions/Manage/Dnn.PersonaBar.Users/Components/RegisterController.cs:287]: I checked all the types and this is the type that best fits the scenario.

The registration email contains the following:

**Subject**
{{Website Name}} New User Registration

**Body**

Dear John Smith,

We are pleased to advise that you have been added as a Registered User to My Website. Please read the following information carefully and be sure to save this message in a safe location for future reference.

Portal Website Address: www.example.com
Username: john.smith@example.com

If you do not know, or cannot remember, your password, please go to http://example.com/default.aspx?ctl=PasswordReset&resetToken=95497741-47c7-4ebd-bcc3-462070526bab to reset it.
Thank you, we appreciate your support...

{{Website Name}}

Comment (sleupold) [reply Extensions/Manage/Dnn.PersonaBar.Users/Components/RegisterController.cs:287]: @Icidis thank you very much for the feedback!

**Review by mitchelsellers — APPROVED** [review]
This change looks good, and respects the existing "Notify" option

**Comment by mitchelsellers** [comment]
@dnnsoftware/ae-approvers Can someone else review this?

**Review by mtrutledge — APPROVED** [review]
looks good

**Comment by valadas** [comment]
@mitchelsellers I tought this was resolved by https://github.com/dnnsoftware/Dnn.Platform/pull/2492 would'nt merging this one too send the notification twice or am I not understanding something here...

**Comment by mitchelsellers** [comment]
Ok, I submitted a PR for the revert if we can get approvals

https://github.com/dnnsoftware/Dnn.AdminExperience/pull/312

**Pull request merged**
